trivago/tas-sdk-php
# WEB-32462 add max_price as input parameter and test

### What
* add max_price as optional parameter of HotelCollectionRequest
* update documentation to include new parameter
* update HotelCollectionRequest unit test to include new parameter

### How to test
* `composer test-quick` to run unit tests
* changing `max_price` parameter should filter the search results
* need to test it against TAS priceRange ticket
